Orange/widgets/regression/owsgdregression.py

```diff
@@ -115,7 +114,10 @@ def __init__(self):
         power_t = gui.doubleSpin(box, self, "power_t", 0.0, 10.0, step=0.01)
         form.addRow("Power t:", power_t)
 
-
+        scroll = QtGui.QScrollArea()
+        scroll.setWidget(box)
+        scroll.setWidgetResizable(True)
+        scroll.setFixedHeight(400)
```

Comment:
I added this by my side as the windows was not fitting on the screen. Should I remove it completely?